.github/workflows/github-actions-prepare-release.yml

```diff
@@ -0,0 +1,81 @@
+name: GitHub Actions - Prepare New Release
+
+on:
+  push:
+    branches:
+      - release/actions
+
+jobs:
+  CheckCreatedBranch:
+    name: Check Created Branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check created release branch
+        uses: actions/github-script@v6
+        with:
+          script: |
+            if ( ! context.payload.created ) {
+              await github.rest.actions.cancelWorkflowRun( {
+                ...context.repo,
+                run_id: context.runId,
+              } );
+            }
+
+  PrepareRelease:
+    name: Prepare Release
+    runs-on: ubuntu-latest
+    needs: CheckCreatedBranch
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Prepare node
+        uses: ./packages/js/github-actions/actions/prepare-node
+        with:
+          node-version: 14
+          cache-dependency-path: ./packages/js/github-actions/package-lock.json
+          install-deps: "no"
+
+      # The checkout revision of this repo itself doesn't have the built actions,
+      # so it needs to build them before using it locally.
+      - name: Build actions
+        run: |
+          cd ./packages/js/github-actions
+          npm ci --ignore-scripts
+          npm run build
+          cd -
+
+      - name: Get release notes
+        id: get-notes
+        uses: ./packages/js/github-actions/actions/get-release-notes
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          package-dir: packages/js/github-actions
+          config-path: packages/js/github-actions/release-notes-config.yml
+          tag-template: "actions-v{version}"
+          minor-keywords: feature, update, enhancement
+
+      - name: Prepare release commits
+        # Use the github-actions bot account to commit.
+        # https://api.github.com/users/github-actions%5Bbot%5D
+        run: |
+          cd ./packages/js/github-actions
+          TODAY=$(date '+%Y-%m-%d')
+          NEXT_VER="${{ steps.get-notes.outputs.next-version }}"
+          CHANGELOG="${{ steps.get-notes.outputs.release-changelog }}"
+          CHANGELOG=$(echo "$CHANGELOG" | sed -E 's/\.? by @[^ ]+ in (https:\/\/github\.com\/.+)/. (\1)/')
```

Comment:
:construction: I think we should also remove the 

```
## New Contributors
```

section. (See https://github.com/woorelease-bugs/initial-release/releases/new?tag=1.0.1 fro an example)

Reply:
The `outputs.release-changelog` output only contains the part of "## What's Changed" heading and its entries. So other contents won't be included. I added output logs in the get-release-notes action by https://github.com/woocommerce/grow/pull/16/commits/dd0226ad09ce3b314615b99ec9ce3660f6ec693c. It should be easier to know the actual values of outputs and to use them.

![image](https://user-images.githubusercontent.com/17420811/174938051-912cac77-a7ad-4a38-a83e-7df336203ae1.png)

Reply:
Ah, ok, now I got it.
`/## What's Changed\n([\d\D]+?)(?=\n\n)/i` catches all the things before the double line break => before 

```md

##New Constributors
```